Compilation error on deploy

Thanks for this – super useful. But I'm having issues deploying. The build is failing on Heroku:

```
Compiling...
Compilation failed:
ModuleNotFoundError: Module not found: Error: Can't resolve 'webpacker-svelte'
```

Tried adding the package as dependency (vs devDependency) but no dice.

Am I doing something stupid?

--- a/README.md
+++ b/README.md
@@ -27,8 +27,8 @@ Once done, run `bundle` to install the gem.
 Then you need to update your `package.json` file to include the `webpacker-svelte` NPM module:
 
 ```bash
-npm i webpacker-svelte --save-dev
-yarn add webpacker-svelte --dev
+npm i webpacker-svelte
+yarn add webpacker-svelte
 ```
 
 You are now all set!

